connect/service.go

```diff
@@ -3,6 +3,8 @@ package connect
 import (
 	"encoding/json"
 	"github.com/micromdm/mdm"
+	apps "github.com/micromdm/micromdm/applications"
```

Comment:
Go package names should always be singular. I'll create an issue once this is merged to update.